### ISSUE TITLE
Add preview support for diffEditor

### ISSUE THREAD

**Issue by fheidrich** [Microsoft/vscode#27308]
- VSCode Version: 1.12.2
- OS Version: OS X 10.11.5

I would like to use preview when opening diffEditor (side-by-side or inline). Right now every file I click on the 'Changes' list opens on a new editor. I rather the files to re-use the same diffEditor.

Ideally this should be configured with a new settings option such as
````
// Controls if the diff editor shows as preview. Preview editors are reused until they are kept (e.g. via double click or editing).
"diffEditor.enablePreview": false, 
````
(default false would keep the current behavior)

Thanks

**Comment by roblourens** [comment]
Related: #18651

**Comment by bpasero** [comment]
I think this was fixed via https://github.com/Microsoft/vscode/issues/26184

**Comment by tibotiber** [comment]
see also #27357

**Comment by joaomoreno** [comment]
Fixed by #26184
